### PR TITLE
EOS 13627 LogRotate Improvement: Remove logrotate custom script and move logrotate conf file at common place

### PR DESCRIPTION
**Pull request by VijayThakkar132**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    EOS 13627: Logrotate improvement
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Remove logrotate custom script and move conf file at common place as provisioner has submitted necessary changes from there side and suggested improvement in the existing logrotation implementation.
  </code>
</pre>
## Solution
<pre>
  <code>
    Removed earlier patch related to logrotate and moved conf file at /etc/logrotate_hourly/.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Verified rpms on the VM.
  </code>
</pre>

### CODE REVIEW

File: conf/script/ha_setup [hunk 1]
```diff
@@ -41,23 +36,13 @@ exec &> >(stdbuf -oL gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0 }')
 post_install() {
     echo "post_install"
     # Add permisson to hacluster
-    add_permission
-
-    # Trigger logotate hourly basis
-    if [ ! -d "${LOGROTATE_DIR_DEST}" ]; then
-        mkdir -p ${LOGROTATE_DIR_DEST}
-    fi
+#    add_permission
```
Comment (indrajitzagade) [inline conf/script/ha_setup:39]: Please delete this line.

Comment (VijayThakkar132) [reply conf/script/ha_setup]: Actually, I had commented out this line for the dev testing as I was getting "hacluster user is not available" error in VM and forgot to uncomment this line after the testing.
Now, Corrected.

Comment (indrajitzagade) [reply conf/script/ha_setup]: ok

**Review by indrajitzagade — APPROVED** [review]

**Review by ajay-paratmandali — APPROVED** [review]

File: conf/script/ha_setup [hunk 2]
```diff
@@ -18,14 +18,9 @@
 HA_PATH="<HA_PATH>"
 SOURCE_HA_CONF="<HA_PATH>/conf/etc"
 HA_CONF="/etc/cortx/ha"
+LOGROTATE_CONF="/etc/logrotate_hourly.d"
```
Comment (ajaysrivas) [inline conf/script/ha_setup:21]: It should be named as LOGROTATE_CONF_DIR. This is a directory, not a config file.

**Review by ajaysrivas — CHANGES_REQUESTED** [review]

File: conf/script/ha_setup (hunk 2 above)
Comment (VijayThakkar132) [reply conf/script/ha_setup]: Done

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**